apps/website/src/content/docs/table.mdx

```diff
@@ -6,16 +6,466 @@ thumbnail: Table
+- `data`: an array containing table data displayed in the table. Each key in each object corresponds to each column's `accessor` property.
+- `columns`: an array containing table column objects. Each column object requires a `Header`, which will be displayed as a column title, and `id`.
+- `emptyTableContent`: a JSX element table content shown when there is no data.
+
+<LiveExample src='Table.basic.jsx'>
+  <AllExamples.TableBasicExample client:load />
+</LiveExample>
+
+**Note**: The `Table` component is built based on `react-table` v7. For more information about `react-table`, see their [docs](https://react-table-v7-docs.netlify.app/).
+
+## Selection
+
+The `Table` component supports row selection, allowing users to select one or more rows for actions by setting the `isSelectable` prop to `true`.
+
+```jsx {6}
+<Table
+  /* … */
+  data={data}
+  columns={columns}
+  emptyTableContent='No data.'
+  isSelectable
+/>
+```
+
+There are two available selection modes passed using the `selectionMode` prop: `"single"` and `"multi"` (default).
+
+### Multi-row selection
+
+In multi-row selection (`selectionMode="multi" | undefined`), a column with checkboxes for toggling a row's selection is added. This mode also supports <kbd>Ctrl</kbd>/<kbd>Command</kbd> + click and <kbd>Shift</kbd> + click shortcuts to select multiple rows.
+
+<LiveExample src='Table.selectMulti.jsx'>
+  <AllExamples.TableSelectMultiExample client:load />
+</LiveExample>
+
+### Single-row selection
+
+In single row selection mode (`selectionMode="single"`), users can select each row by clicking on the desired row.
+
+**Note**: For accessibility reasons, in single-selection Tables, it is highly recommended to include a button in the primary cell of each row. [More info](https://github.com/iTwin/iTwinUI/issues/2130#issuecomment-2209287280).
+
+```jsx
+const columns = React.useMemo(
+  () =>
+    [
+      /* … */
+      {
+        /* … */
+        Cell: ({ value }) => {
+          return (
+            <Anchor
+              as="button"
+              onClick={() => console.log(`Selected ${value}`)}
+            >
+              {value}
+            </Anchor>
+          );
+        },
+      },
+    ] satisfies Column[],
+  []
+);
+```
+
+<LiveExample src='Table.selectSingle.jsx'>
+  <AllExamples.TableSelectSingleExample client:load />
+</LiveExample>
+
+### Prevent accidental row selection
+
+By default, `selectRowOnClick` is `true`. This is ideal for most cases. However, this can be disabled in use-cases where the loss of the selection state due to an accidental row click is too disruptive.
+
+```jsx {3}
+<Table
+  /* … */
+  selectRowOnClick={false}
+/>
+```
+
+To prevent the selection of the row when just hoping to select the cell content, we add a wrapper to increase the hit size of the cell content. However, this does not happen if a `cellRenderer` is passed with custom children.
```

Comment:
Avoid using "we" in documentation.

```suggestion
To prevent the selection of the row when just hoping to select the cell content, iTwinUI adds a wrapper to increase the hit size of the cell content. However, this does not happen if a `cellRenderer` is passed with custom children.
```

Reply:
The docs are about iTwinUI, not about us. I would recommend reading this if you want to understand how to write better documentation: https://contribute.docs.astro.build/guides/writing-style/